atomic_reactor/tasks/common.py

```diff
@@ -90,6 +93,8 @@ class Task(abc.ABC, Generic[ParamsT]):
     def __init__(self, params: ParamsT):
         """Initialize a Task."""
         self._params = params
+        self.data = self.load_workflow_data()
```

Comment:
Possibly a pro-tip for the future: fixing unit tests might have been less painful if this was a cached property. Currently, you need to mock context data just to initialize any Task. With a property, you'd only have to mock in the tests that actually use the data, which would probably be more or less the same tests that already do the mocking.

(No need to change it now)

Reply:
hmm, would that not have detrimental effect when we want to actually update the property?

Reply:
I always welcome pro-tips, updated.